site/source/docs/porting/exceptions.rst

```diff
@@ -74,3 +74,18 @@ throwing and catching exceptions to WebAssembly.
 As a result, it can reduce code size and performance overhead compared
 to the JavaScript-based implementation. However, it's still brand-new
 and `not yet supported by default in most engines <https://webassembly.org/roadmap/>`_.
+
+JavaScript / C++ interaction
+############################
+
+No effort is made to bridge the JS and C++ worlds in terms of catching
+exceptions from the other side and automatically converting them. That is, you
+cannot expect a C++ try-catch to catch a JS exception. C++ can only catch C++
```

Comment:
Correct me if I'm wrong but I believe there are two big points to make here:
- C++ can't catch JS exceptions
- The C++ stack is not unwound correctly if a JS exception bubbles up through it. Destructors are not called

I'd argue that the second one is the really scary one and we should make it super clear to anyone using emscripten EH